### PR TITLE
FLOE-402: repo setup

### PR DESCRIPTION
**Pull request by jobara**
Setting up the repo with the basic required files including license, README, and linting

https://issues.fluidproject.org/browse/FLOE-402

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    "authoring",
+    "web",
+    "fluid",
+    "infusion",
+    "sonfication"
+  ],
+  "repository": "git://github.com/fluid-project/chartAuthoring.git",
+  "main": "",
+  "engines": {
+    "node": ">=0.1.9"
```
Comment (amb26) [inline package.json]: It's unlikely that we will support any version of node earlier than 0.10.x

File: README.md [hunk 2]
```diff
@@ -1,2 +1,15 @@
-# chartAuthoring
-A tool for authoring accessible charts for the web.
+##What Is the Chart Authoring Tool?##
+
+A tool for authoring accessible charts for the web. Including a sonified (audio) version of the chart.
+
+##How Do I Get Infusion?##
```
Comment (amb26) [inline README.md]: Stale heading

Comment (cindyli) [inline README.md]: Change "Infusion" to "the Chart Authoring Tool".

File: README.md [hunk 3]
```diff
@@ -1,2 +1,15 @@
...
+
+You can checkout and fork the Chart Authoring Tool on github:
+
+<https://github.com/fluid-project/chartAuthoring>
+
+##Who Makes Infusion, and How Can I Help?##
```
Comment (cindyli) [inline README.md]: Change "Infusion" too.

File: package.json [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+  },
+  "devDependencies": {
+    "grunt": "~0.4.2",
```
Comment (cindyli) [inline package.json]: The latest stable grunt version is 0.4.5

**Comment by jobara** [comment]
@cindyli and @amb26 this is ready for another round of review

**Comment by cindyli** [comment]
Merged @ c67342cbedb0d38637c6f46e4f58457eaf487568

**Pull request merged**